ups/ap_verify.table

```diff
@@ -9,8 +9,12 @@ setupRequired(verify)
 setupRequired(ap_pipe)
 
 setupOptional(ap_verify_testdata)
-setupOptional(ap_verify_hits2015)
 setupOptional(obs_test)
 
+# Datasets other than ap_verify_testdata must be set up manually
+#     to avoid being inadvertently downloaded by lsstsw
+# setupOptional(ap_verify_hits2015)
```

Comment:
I don't like having executable code commented out.  Maybe instead just list the packages you could use?  I.e.
`# setupOptional(ap_verify_hits2015)` --> `#      ap_verify_hits2015`